website/cue/reference/components/sources/journald.cue

```diff
@@ -92,6 +120,34 @@ components: sources: journald: {
 				}
 			}
 		}
+		include_matches: {
+			common:      true
+			description: "This list contains sets of field/value pairs to monitor. If empty or not present, all journal fields are accepted. If `include_units` is specified, it will be merged into this list."
+			required:    false
+			warnings: []
+			type: object: {
+				examples: [
+					{
+						_SYSTEMD_UNIT: ["sshd.servcie", "ntpd.service"]
+						_TRANSPORT: ["kernel"]
+					},
+				]
+				options: {
+					"*": {
+						common:		 false
+						description: "The set of field values to match in journal entries that are to monitor."
```

Comment:
Almost. 
```suggestion
						description: "The set of field values to match in journal entries that are to be included."
```

Reply:
Thank you for the feedback!
fix 4480b5d